src/libmongoc/tests/json-test.c

```diff
@@ -1599,6 +1599,12 @@ set_auto_encryption_opts (mongoc_client_t *client, bson_t *test)
       bson_free (env_cryptSharedLibPath);
    }
 
+   if (bson_iter_init_find (&iter, &opts, "keyExpirationMS")) {
+      BSON_ASSERT (BSON_ITER_HOLDS_INT (&iter));
+      const int expiration = bson_iter_int32 (&iter);
+      mongoc_auto_encryption_opts_set_key_expiration (auto_encryption_opts, expiration);
```

Comment:
Suggest using `bson_iter_as_int64`. `bson_iter_int32` returns 0 if the BSON type is int64:

```c
BSON_ASSERT (BSON_ITER_HOLDS_INT (&iter));
const int64_t expiration = bson_iter_as_int64 (&iter);
BSON_ASSERT (expiration > 0);
mongoc_auto_encryption_opts_set_key_expiration (auto_encryption_opts, (uint64_t) expiration);
```